site/src/sphinx/server-sse.rst

```diff
@@ -0,0 +1,49 @@
+.. _`Server-sent Events`: https://www.w3.org/TR/eventsource/
+
+.. _server-sse:
+
+Serving Server-sent Events
+==========================
+
+A traditional web page has to send a request to the server in order to receive new data.
+With `Server-sent Events`_, however, it is possible for a server to push a new data to the web page
+whenever it wants to.
+
+Armeria provides several factory methods by :api:`ServerSentEvents` class which help a user to easily send
+an event stream to the client. The following example shows how to build a server which serves services
+sending a response with `Server-sent Events`_.
+
+.. code-block:: java
+
+    import com.linecorp.armeria.common.sse.ServerSentEvent;
+    import com.linecorp.armeria.server.Server
+    import com.linecorp.armeria.server.ServerBuilder
+    import com.linecorp.armeria.server.streaming.ServerSentEvents;
+
+    Server server = new ServerBuilder()
+            // Emit Server-sent Events with the SeverSentEvent instances published by a publisher.
+            .service("/sse1",
+                     (ctx, req) -> ServerSentEvents.fromPublisher(
+                             Flux.just(ServerSentEvent.ofData("foo"), ServerSentEvent.ofData("bar"))))
+            // Emit Server-sent Events with converting instances published by a publisher into
+            // ServerSentEvent instances.
+            .service("/sse2",
+                     (ctx, req) -> ServerSentEvents.fromPublisher(
+                             Flux.just("foo", "bar"), ServerSentEvent::ofData))
+            .build();
+
+Of course, Armeria provides :api:`@ProducesEventStream` annotation in order to convert the result objects
+returned from an annotated service method into `Server-sent Events`_. The following example shows how to
+use the annotation.
+
+.. code-block:: java
+
+    import com.linecorp.armeria.common.sse.ServerSentEvent;
```

Comment:
Could remove this import statement.

Reply:
😃 It's using when creating `ServerSentEvent` instance:
`ServerSentEvent.ofData("foo")`